### ISSUE TITLE
Add contract selection to time registration fieldset when editing issue

### ISSUE THREAD

**Issue by martijnv** [upgradeya/redmine-contracts-with-time-tracking-plugin#48]
Great module, it looks like it could really be useful to use.

What we notices is that when you edit an issue you also have a time registration fieldset where one can log time. However, there is no contract selection possible in that fieldset.

You might want to add the contract selection to that fieldset or disable/hide the fieldset if that is not possible.

**Comment by partyp** [comment]
Good catch and reminder. This was something we wanted from the beginning, but I believe there was no hook for us to use the "log time" fieldset within an issue.  I'm not sure if we can disable that fieldset either. Will look into it.
